package.json

```diff
@@ -29,6 +29,11 @@
     "bundle-loader": "^0.5.2",
     "chai": "^3.5.0",
     "css-loader": "^0.9.1",
+    "eslint": "^4.5.0",
+    "eslint-config-airbnb": "^15.1.0",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-jsx-a11y": "^5.1.1",
+    "eslint-plugin-react": "^7.3.0",
```

Comment:
These last three here look unused

Reply:
@KamranAsif These are peerDependencies for `eslint-config-airbnb`.
Comment from the [npm site](https://www.npmjs.com/package/eslint-config-airbnb):
> It requires eslint, eslint-plugin-import, eslint-plugin-react, and eslint-plugin-jsx-a11y.